AMBARI-23672: Change the method identifier "extractHeaders" to "getHeaders".

Kui-Liu opened the pull request:
The method just returns the field "headers" of this class, thus method name "getHeaders" should be better than "extractHeaders".

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/1882/
Test PASSed.

Kui-Liu committed:
AMBARI-23672: Change two method names.

The two methods are named as "extractXXX", but they just return a LinkedList value.
Just naming it as "getXXX" should be more clear since "extractXXX" is prone to mean extracting something from a dataset.

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/1961/
Test FAILed.
Test FAILured.

adoroszlai commented:
These files have been removed in b623f07013, so this change is no longer relevant.
